packages/web-components/src/components/va-accordion/va-accordion.tsx

```diff
@@ -27,6 +27,10 @@ import { getSlottedNodes } from '../../utils/utils';
  * - Allow users to click anywhere in the header area to expand or collapse the content; a larger target is easier to manipulate.
  * - Make sure interactive elements within the collapsible region are far enough from the headers that users don’t accidentally trigger a collapse. (The exact distance depends on the device.
  */
+
+/**
+ * @accordionItemToggled This event is fired when an accordion item is opened or closed
```

Comment:
This looks like it does bubble up out of the wrapper so I think it's fine to advertise it like this. However, I noticed in the subcomponent arg table it isn't ending up in the stacked `Events` table:

![image](https://user-images.githubusercontent.com/2008881/152024284-d89542d8-a4b1-411c-b459-b4befe4b17f1.png)

I don't think this should block a merge, but if this can be like a 20 minute fix it would be nice to include it. If `subcomponents` work differently and there's nothing we can do about it then that's fine :shrug:

Reply:
It looks like this might not be possible for subcomponents. I found this: https://github.com/storybookjs/storybook/issues/14710